ndcube/mixins/plotting.py

```diff
@@ -371,6 +371,16 @@ def _animate_cube_1D(self, plot_axis_index=-1, axes_coordinates=None,
             else:
                 unit_x_axis = None
         # Put xdata back into axes_coordinates as a masked array.
+
```

Comment:
Add a comment here saying something like: ```Since LineAnimator currently only accepts 1-D arrays for the x-axis, collapse xdata to single dimension by taking mean along non-plotting axes.```

Reply:
Sure.